app/components/search_bar/search_bar.ios.js

```diff
@@ -116,9 +116,9 @@ export default class SearchBarIos extends Component {
                     {...this.props}
                     ref='search'
                     placeholderCollapsedMargin={this.state.placeholderWidth}
-                    placeholderExpandedMargin={20}
-                    searchIconCollapsedMargin={this.state.placeholderWidth + 10}
-                    searchIconExpandedMargin={10}
+                    placeholderExpandedMargin={30}
```

Comment:
Tried with this values

```js
placeholderCollapsedMargin={this.state.placeholderWidth - 15}
                    placeholderExpandedMargin={30}
                    searchIconCollapsedMargin={this.state.placeholderWidth}
                    searchIconExpandedMargin={15}
```

and that looks like this on an iPhone 5s
![image](https://user-images.githubusercontent.com/6757047/27136649-dea7e26e-50e9-11e7-96db-7191d6445b13.png)